src/views/auth/CreateNewGame.js

```diff
@@ -62,8 +65,22 @@ class CreateNewGame extends React.Component {
     });
   }
 
+  handleNewUserMessage(msg){
+    let acceptedUsers = msg.users.map(acceptedUser => {
+      return createUser(acceptedUser.username, "friend@friend.ch", null, userCategories.ACCEPTED)
+    })
+
+    this.setState({currentPlayers: acceptedUsers});
```

Comment:
currentPlayers is shared among invitedUsers and acceptedUsers, if we set the state here to only the acceptedUsers we delete all invitedUsers from the list. We should only update the accepted users here (see also handleInvitedUserMessage())